packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -77,17 +78,97 @@ enum SystemUiOverlay {
   bottom,
 }
 
+/// Specifies a dark vs light preference for certain system chrome options.
+enum SystemChromeTheme {
+  /// Whether the setting should be drawn to be compatible with a light color.
+  light,
+  /// Whether the setting should be drawn to be compatible with a dark color.
+  dark,
+}
+
 /// Specifies a preference for the style of the system overlays.
 ///
 /// Used by [SystemChrome.setSystemUIOverlayStyle].
-enum SystemUiOverlayStyle {
+class SystemUiOverlayStyle {
   /// System overlays should be drawn with a light color. Intended for
   /// applications with a dark background.
-  light,
+  static const SystemUiOverlayStyle light = const SystemUiOverlayStyle(
+    navigationBarColor: 0xFF000000,
+    navigationDividerColor: null,
+    statusBarColor: null,
+    navigationIconTheme: SystemChromeTheme.light,
+    statusBarTheme: SystemChromeTheme.light,
+  );
 
   /// System overlays should be drawn with a dark color. Intended for
   /// applications with a light background.
-  dark,
+  static const SystemUiOverlayStyle dark = const SystemUiOverlayStyle(
+    navigationBarColor: 0xFFFFFFFF,
```

Comment:
White seems at odds with `overlays should be drawn with a dark color`. 

Shouldn't this version specify a `navigationDividerColor` (because a non-white divider looks best on white).

Reply:
Fixed colors.

I can add a divider color, but we can't actually invoke that API until we update our target SDK version to P though.  We can address this later if we decide to draw the bar ourselves using a transparent navigation bar